### PR TITLE
using redis

### PR DESCRIPTION
**Pull request by kalantar**

### CODE REVIEW

File: docs/tutorials/integrations/kserve/http.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ kubectl delete inferenceservice sklearn-irisv2
 If you installed Grafana, you can delete it as follows:
 
 ```shell
-kubectl delete svc/grafana, deploy/grafana
+kubectl delete svc/grafanas deploy/grafana
```
Comment (Alan-Cha) [inline docs/tutorials/integrations/kserve/http.md:100]:
```suggestion
kubectl delete svc/grafana deploy/grafana
```

File: docs/user-guide/topics/metrics_store.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+# Metrics store
+
+One of Iter8's key advantages is that it incorporates its own metrics store simplifying the set up and execution of A/B/n and performance tests. This metrics store, based on BadgerDB is not suitable for production use. It is suitable only for a single instance of Iter8.
+
+To support production use cases, the metrics store can be replaced with another distributed database such as Redis. Currently Iter8 supports:
+
+- BadgerDB
+- Redis
```
Comment (Alan-Cha) [inline docs/user-guide/topics/metrics_store.md:12]: I think this section should first mention the supported databases, and then mention which are suitable for production use cases.

For example:

One of Iter8's key advantages is that it incorporates its own metrics store, simplifying the set up and execution of A/B/n and performance tests. Iter8 currently supports the following databases:

- BadgerDB
- Redis

Note that BadgerDB is not suitable for production use and is only suitable for a single instance of Iter8. Support for other databases are in the works. See [below](#contribute-a-new-metrics-store-implementation) for details on how to contribute additional implementations.

File: docs/user-guide/topics/metrics_store.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+See [below](#contribute-a-new-metrics-store-implementation) for details on how to contribute additional implementations.
+
+## Using Redis as the metrics store 
+
+We assume that Redis is deployed. For example, for a basic deployment:
```
Comment (Alan-Cha) [inline docs/user-guide/topics/metrics_store.md:18]: Is there a need to state that we use BadgerDB by default somewhere?

File: docs/user-guide/topics/metrics_store.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+```shell
+kubectl create deploy redis --image=redis/redis-stack:latest --port=6379
+kubectl expose deploy redis --port=6379
+```
+
+Run Iter8 identifying the metrics store implementation as `redis` and identify the endpoint:
```
Comment (Alan-Cha) [inline docs/user-guide/topics/metrics_store.md:25]:
```suggestion
Run Iter8 with metrics store implementation as `redis` and the respective endpoint:
```

File: docs/user-guide/topics/metrics_store.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+    - Update [go.mod](https://github.com/iter8-tools/iter8/blob/master/go.mod) and [go.sum](https://github.com/iter8-tools/iter8/blob/master/go.sum) if needed.
+
+    - Update the default Helm chart configuration [values.yaml](https://github.com/iter8-tools/iter8/blob/master/charts/controller/values.yaml) and bump the chart version in [Chart.yaml](https://github.com/iter8-tools/iter8/blob/master/charts/controller/Chart.yaml).
+
+    - Please also consider including your information in our list of [adopters](https://github.com/iter8-tools/iter8/blob/master/ADOPTERS.md).
+
+3. Submit a second pull request on the Iter8 [docs project](https://github.com/iter8-tools/docs) updating the list of available implementations.
+
+4. Alert the project reviewers on [slack](https://join.slack.com/t/iter8-tools/shared_invite/zt-awl2se8i-L0pZCpuHntpPejxzLicbmw) `#development` channel.
```
Comment (Alan-Cha) [inline docs/user-guide/topics/metrics_store.md:54]: Is this the right place to put this? I think it makes sense to put this into the `CONTRIBUTING.md`. Thoughts?

Comment (Alan-Cha) [inline docs/user-guide/topics/metrics_store.md:54]:
```suggestion
## Contribute a new metrics store implementation

To contribute a new metrics store implementation:

1. Create an [issue](https://github.com/iter8-tools/iter8/issues) for discussion.

2. Submit a pull request on the [Iter8 project](https://github.com/iter8-tools/iter8) with the following updates:

    - Create sub-folder in [storage](https://github.com/iter8-tools/iter8/tree/master/storage) and provide an implementation of this [interface](https://github.com/iter8-tools/iter8/blob/master/storage/interface.go) including test cases.

    - Add a new case to [metrics.GetClient()](https://github.com/iter8-tools/iter8/blob/master/metrics/client.go).

    - Update [go.mod](https://github.com/iter8-tools/iter8/blob/master/go.mod) and [go.sum](https://github.com/iter8-tools/iter8/blob/master/go.sum) if needed.

    - Update the default Helm chart configuration [values.yaml](https://github.com/iter8-tools/iter8/blob/master/charts/controller/values.yaml) and bump the chart version in [Chart.yaml](https://github.com/iter8-tools/iter8/blob/master/charts/controller/Chart.yaml).

    - Please also consider including your information in our list of [adopters](https://github.com/iter8-tools/iter8/blob/master/ADOPTERS.md).

3. Submit a second pull request on the Iter8 [docs project](https://github.com/iter8-tools/docs) updating the list of available implementations.

4. Alert the project reviewers on [Slack](https://join.slack.com/t/iter8-tools/shared_invite/zt-awl2se8i-L0pZCpuHntpPejxzLicbmw) `#development` channel.
```

File: docs/user-guide/topics/metrics_store.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+## Contribute a new metrics store implementation
+
+To contribute a new metrics store implementation:
+
+1. Create an [issue](https://github.com/iter8-tools/iter8/issues) for discussion.
+
+2. Submit a pull request on the [Iter8 project](https://github.com/iter8-tools/iter8) with the following updates:
+
+    - Create sub-folder in [storage](https://github.com/iter8-tools/iter8/tree/master/storage) and provide an implementation of this [interface](https://github.com/iter8-tools/iter8/blob/master/storage/interface.go) including test cases.
```
Comment (Alan-Cha) [inline docs/user-guide/topics/metrics_store.md:42]: I recall that the BadgerDB and redis implementations are both named `simple.go`. This was just an artifact of when we were first toying around with the idea of using a database. I think if we are going to make users aware that they can contribute, we should also change this file name to a more sensible name.

File: docs/user-guide/topics/metrics_store.md (hunk 5 above)
Comment (kalantar) [reply docs/user-guide/topics/metrics_store.md:54]: I think this is the right place. It is specific to adding a metrics storage implementation. Some repeats what is in CONTRIBUTING.md but I thought for clarity it made sense to add here as well.

File: docs/user-guide/topics/metrics_store.md (hunk 3 above)
Comment (kalantar) [reply docs/user-guide/topics/metrics_store.md:18]: add to introduction

**Review by Alan-Cha — DISMISSED** [review]

File: docs/user-guide/topics/metrics_store.md [hunk 7]
```diff
@@ -4,14 +4,12 @@ template: main.html
...
-One of Iter8's key advantages is that it incorporates its own metrics store simplifying the set up and execution of A/B/n and performance tests. This metrics store, based on BadgerDB is not suitable for production use. It is suitable only for a single instance of Iter8.
-
-To support production use cases, the metrics store can be replaced with another distributed database such as Redis. Currently Iter8 supports:
+One of Iter8's key advantages is that it incorporates its own metrics store simplifying the set up and execution of A/B/n and performance tests. Iter8 currently supports the following databases:
 
 - BadgerDB
 - Redis
 
-See [below](#contribute-a-new-metrics-store-implementation) for details on how to contribute additional implementations.
+Iter8 uses BadgerDB by default. Note, however, that BadgerDB is not suitable for prodcution use and is only suitable for a single instance of Iter8. Support for other databases are in the works. See [below](#contribute-a-new-metrics-store-implementation) for details on how to contribute additional implementations.
```
Comment (Alan-Cha) [inline docs/user-guide/topics/metrics_store.md:12]:
```suggestion
Iter8 uses BadgerDB by default. Note, however, that BadgerDB is not suitable for production use and is only suitable for a single instance of Iter8. Support for other databases are in the works. See [below](#contribute-a-new-metrics-store-implementation) for details on how to contribute additional implementations.
```

**Review by Alan-Cha — APPROVED** [review]

**Pull request merged**